keyboards/mechlovin/olly/jf/keymaps/via/config.h

```diff
@@ -0,0 +1,3 @@
+#define DYNAMIC_KEYMAP_EEPROM_MAX_ADDR 1607
```

Comment:
This needs a license header, and an include guard (`#pragma once`)

Reply:
yes, thank you.